crates/bevy_asset_store/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! A asset store for Bevy Editor.
```

Comment:
I think we should have a clearer vision of how the editor interacts with the store, and what this crate handles.

I'd be interested in functionality for checking for updates to assets, and breadcrumbs to the website to browse assets, but at least for now I'd like to keep the data and interface totally on the website to avoid having to duplicate efforts.

Reply:
My personal interpretation is the crate acts as a link between the website asset store and the editor's asset browser pane, implementing things like loading previews of assets, essential it'd be full of networking code to connect the editor to the store. I figured it best to be separated from the asset browser so if someone wants they can easily cut out the asset store no problem.

Reply:
Hmm, I see! Can you add a `bevy_asset_preview` crate as part of this PR and explain the relationship between the two in the module docs here?